Debian: missing JSON dependency

If you try to install `fingerbank` without `packetfence` on a fresh Debian 9 installation, you will got following error:
```
Can't locate JSON.pm in @INC (you may need to install the JSON module) (@INC contains: /usr/local/fingerbank/lib /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.24.1 /usr/local/share/perl/5.24.1 /usr/lib/x86_64-linux-gnu/perl5/5.24 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.24 /usr/share/perl/5.24 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at /usr/local/fingerbank/lib/fingerbank/DB.pm line 16.
BEGIN failed--compilation aborted at /usr/local/fingerbank/lib/fingerbank/DB.pm line 16.
Compilation failed in require.
BEGIN failed--compilation aborted.
Makefile:128: recipe for target 'fixpermissions' failed
make: *** [fixpermissions] Error 2
dpkg: error processing package fingerbank (--configure):
 subprocess installed post-installation script returned error exit status 2
Processing triggers for libc-bin (2.24-11+deb9u4) ...
Errors were encountered while processing:
 fingerbank
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

Issue is caused by the fact that `libjson-perl` is not a dependency of `fingerbank`.

### Workaround

`apt install libjson-perl`

--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,9 @@
+fingerbank (4.1.3-3) unstable; urgency=medium
+
+  * Allow fingerbank to be installed without packetfence on Debian
+
+ -- Inverse inc. <info@inverse.ca>  Fri, 28 Jun 2019 14:28:17 +0200
+
 fingerbank (4.1.3-2) unstable; urgency=low
 
   * Adapt Debian packaging to CI

--- a/debian/control
+++ b/debian/control
@@ -19,6 +19,7 @@ Depends: ${misc:Depends},
  libdbd-sqlite3-perl, liblwp-protocol-https-perl, sqlite3,
  libdata-powerset-perl,
  liblwp-protocol-connect-perl,
+ libjson-perl,
  libsql-translator-perl, libfile-touch-perl, libconfig-inifiles-perl (>= 2.88),
  fingerbank-collector (>= 1.0.1),
 Description: fingerbank

--- a/debian/fingerbank.postinst
+++ b/debian/fingerbank.postinst
@@ -30,7 +30,6 @@ case "$1" in
         make init-db-local
         make fixpermissions
         /usr/local/fingerbank/conf/upgrade/*
-        chown pf.pf /usr/local/fingerbank/conf/fingerbank.conf
     ;;
 
     abort-upgrade|abort-remove|abort-deconfigure)

